MacPorts/settings.py

```diff
@@ -147,6 +147,7 @@
 # Configuration for CORS
 CORS_ORIGIN_WHITELIST = [
     'http://localhost:8010',
+    'http://34.67.147.76',
```

Comment:
In a way I would prefer if this commit only went to the branch that gets deployed (and if that special branch would be pushed to this repository). The settings are just temporary.

Reply:
Okay, thank you. Currently, this branch (week3/improvements) gets deployed.

To clarify, the special branch you are talking about should also be pushed here?

Reply:
over to #27